### ISSUE TITLE
Ingress gateway does not respect the values.global.proxy.accessLogEncoding argument

### ISSUE THREAD

**Issue by taylorreece** [istio/istio#19763]
**Bug description**
When installing istio via Helm, if you set `global.proxy.accessLogFile=/dev/stdout` and `global.proxy.accessLogEncoding=JSON`, your ingress gateway spits out JSON output.  That's desirable.

However, if you install via `istioctl`, you do not get JSON output.  

If you install via `istioctl` and then manually patch in `accessLogEncoding: JSON` just after `accessLogFile: "/dev/stdout"` with `kubectl edit cm istio -n istio-system`, and then kick the ingress gateway pod, your desired JSON output returns.

**Expected behavior**
JSON output from the ingress gateway

**Steps to reproduce the bug**
Install a cluster with `./bin/istioctl manifest apply --set values.global.proxy.accessLogFile=/dev/stdout --set values.global.proxy.accessLogEncoding=JSON` and then get logs from the ingress gateway; they are not JSON-formatted.

**Version (include the output of `istioctl version --remote` and `kubectl version` and `helm version` if you used Helm)**
```
$ istioctl version
client version: 1.4.2
control plane version: 1.4.2
data plane version: 1.4.2 (1 proxies)
```
**How was Istio installed?**
Via istioctl (see steps to "reproduce the bug")

**Environment where bug was observed (cloud vendor, OS, etc)**
GCP and local docker/kubernetes

**Comment by taylorreece** [comment]
Will https://github.com/istio/installer/commit/715ec75c7ea4bddd61d3a68c8a8e8ad047987eaa perhaps solve this issue?  I'm a bit perplexed by how the different https://github.com/istio/* projects work together.

**Comment by howardjohn** [comment]
Yes, that will fix it so it will be fixed on 1.4.3 so I think there is nothing to do here.

**Comment by taylorreece** [comment]
:-( Looks like it missed the 1.4.3 cut-off.  Oh, well.  🤞 for 1.4.4 :-)
